### ISSUE TITLE
A11y_SQLAzureDataStudio_DeployDacpac_DeployDacpacSettings_Screenreader: Jaws doesn’t announce that Target Server and Database Name Combobox as required fields

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9326]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1#Screenreader;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Additional Details:
MAS References: MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select first radio button.
9. Navigate till next button & hit enter.
10. Launch JAWS screenreader
11. Navigate to Target Server and Database Name Combobox using down arrow key

**Actual:**
Jaws doesn’t announce that Target Server and Database Name Combobox as required fields.

**Expected:**
Jaws should announce that Target Server and Database Name Combobox as required fields.

**Attachment link for Reference:**
[9326_A11y_SQLAzureDataStudio_DeployDacpac_DeployDacpacSettings_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274794/9326_A11y_SQLAzureDataStudio_DeployDacpac_DeployDacpacSettings_Screenreader.zip)

**Comment by kisantia** [comment]
Duplicate of #9183. Fixed by #9200

**Comment by kapilvaishna** [comment]
@kisantia could you please reopen the bub as issue is partially solved as it is not announcing required for Database Name Combo box
**GithubTags**:#Regressed:08-02-23;

https://user-images.githubusercontent.com/124254981/217545727-a6e54c95-35ec-4a91-8c0a-2dd2c7717453.mp4

**Comment by kisantia** [comment]
@kapilvaishna the fix has been released in the dacpac extension version 1.13.0 in the [ADS insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch). Please verify this is fixed. Thanks!

**Comment by kapilvaishna** [comment]
Issue does not repro in environment dacpac extension version 1.13.0 in the ADS insiders build.
So, closing the bug
@kisantia bug is verified you can close.

**GithubTags**:#Closed;
**GithubTags**:#Regressed:17

https://user-images.githubusercontent.com/124254981/219645570-31c532c7-faa6-4e81-a6a3-f0589fb2cd5a.mp4

-02-23;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
